hoomd/operations.py

```diff
@@ -75,29 +150,43 @@ def _store_reader(self, reader):
         # TODO
         pass
 
-    def __contains__(self, obj):
-        return any(op is obj for op in self)
+    def __contains__(self, operation):
+        """Whether an operation is contained in this container.
+
+        Args:
+            operation (``any``): Returns whether this exact operation is
+                contained in the collection.
+        """
+        return any(op is operation for op in self)
 
     def __iter__(self):
-        if self._integrator is not None:
-            yield from chain((self._integrator,), self._analyzers,
-                             self._updaters, self._tuners)
-        else:
-            yield from chain((self._analyzers, self._updaters, self._tuners))
+        """Iterates through all contained operations."""
+        integrator = (self._integrator) if self._integrator else []
+        yield from chain(
+            integrator, self._analyzers, self._updaters, self._tuners)
```

Comment:
In #574, we should adjust this order to match the order in which the operations will be evaluated during the run loop.